server/stt/stt_events_ml.py

```diff
@@ -86,6 +104,20 @@ def parse(self, tree: Element, provider=None):
                 # If the item contains the ``sttinstruct:remove`` signal, no need to ingest this one
                 continue
             self.set_extra_fields(item, tree)
+
+            fields = (
```

Comment:
This functionality is already in EventsML parser from `superdesk-planning`. And we have to check if the field is using `Editor3`, otherwise adding HTML tags in plain text would not be ideal.

As we support this as a core feature, we should update the EventsML and PlanningML parsers to support all possible fields. See https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/feed_parsers/events_ml.py#L252 for example.

We first check if the field is Editor3 in the content profile, and then attempt to convert to html. So it might be good to implement a utility in `superdesk-planning` to convert multiple fields on ingest to editor3, checking it's content profile.